libnavui-dropin/src/test/java/com/mapbox/navigation/dropin/component/maneuver/ManeuverProcessorTest.kt

```diff
@@ -0,0 +1,130 @@
+package com.mapbox.navigation.dropin.component.maneuver
+
+import com.mapbox.bindgen.ExpectedFactory
+import com.mapbox.navigation.base.trip.model.RouteProgress
+import com.mapbox.navigation.dropin.component.navigationstate.NavigationState
+import com.mapbox.navigation.ui.maneuver.api.MapboxManeuverApi
+import com.mapbox.navigation.ui.maneuver.model.Maneuver
+import com.mapbox.navigation.ui.maneuver.model.ManeuverError
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Test
+
+class ManeuverProcessorTest {
+
+    @Test
+    fun processVisibility_navigationState_empty_maneuverIsError() {
+        val result = ManeuverProcessor.ProcessVisibility(
+            NavigationState.Empty,
+            ExpectedFactory.createError(mockk<ManeuverError>())
+        ).process()
+
+        assertFalse(result.isVisible)
+    }
+
+    @Test
+    fun processVisibility_navigationState_empty_maneuverIsValue() {
```

Comment:
nit: junit5 looks like it is trying to parameterize tests more. but sealed classes may have issues. was looking around and found this https://github.com/dpreussler/junit5-kotlin but not sure we really want to integrate that

short term maybe we could just do it ourselves. my tests aren't working though

 ``` kotlin
    @Test
    fun errors_are_not_visible() {
        val navigationStates = NavigationState::class.sealedSubclasses
            .map { it.objectInstance as NavigationState }
        navigationStates.onEach { navigationState ->
            val result = ManeuverProcessor.ProcessVisibility(
                navigationState,
                ExpectedFactory.createError(mockk())
            ).process()

            assertFalse("$navigationState", result.isVisible)
        }
    }
```